import requests.py

```diff
@@ -0,0 +1,86 @@
+import requests
+import json
+import time
+import hmac
+import hashlib
+import pandas as pd
+import os
+
+
+def to_csv(df, csv_name):
+    path = f"/Users/elab/Downloads/{csv_name}.csv"
+    df.to_csv(path, mode='a', header=not os.path.exists(path), index=False)
+
+def create_signature(query_string, secret):
+    return hmac.new(secret.encode('utf-8'), query_string.encode('utf-8'), hashlib.sha256).hexdigest()
+
+def Option_Account_Information(api_key, api_secret):
+    base_url = 'https://api.binance.com'
+    endpoint = '/eapi/v1/marginAccount'
+    api_key = 'vUUfFsfXMzJkZTaVpE84XZDMxIrHKkPoLtrRTR1rLzth6lZMQXjZE8ifokMjk6Mo'
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded API key. This exposes your system to a higher risk of unauthorized access, as anyone who gets hold of this key can potentially misuse your API and its associated data. Possible remediation approaches are:
	 1. Manage secret data using environment variables:
		 a) Identify the secret data and store that data in environment variables on your local and production server.
		 b) Replace the secrets with references to environment variables.
	 2. Use AWS Secrets Manager to store and access secrets:
		 a) Visit Secrets Manager on the AWS Management Console.
		 b) Choose the secret type on the console page and follow the instructions.
		 c) Use the suggested code samples to retrieve the secret.